packages/react/src/conditional-reveal/ConditionalReveal.test.tsx

```diff
@@ -0,0 +1,69 @@
+import '@testing-library/jest-dom';
+import 'html-validate/jest';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { useState } from 'react';
+import {
+	act,
+	cleanup,
+	fireEvent,
+	render,
+	screen,
+} from '../../../../test-utils';
+import type { ConditionalRevealProps } from './ConditionalReveal';
```

Comment:
This is our standard
```suggestion
import { type ConditionalRevealProps } from './ConditionalReveal';
```

Reply:
Merged both imports for `./ConditionalReveal` in [9694d0](https://github.com/agriculturegovau/agds-next/pull/1984/commits/9694d02d980b5f4433a32c7d871467a67245c2e4)